src/components/Header/Header.js

```diff
@@ -1,5 +1,78 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
 export default function Header() {
-  return <div>Header</div>;
+  return (
+    <HeaderWrap>
+      <Inner>
+        <LogoLink to="/">
+          <img src="/images/header/logo.png" alt="BugBug" />
+        </LogoLink>
+
+        <div>
+          <BtnUpload className="btn upload">
+            <span>프로젝트 올리기 시작하기</span>
```

Comment:
```suggestion
            <BtnTitle title="upload">프로젝트 올리기 시작하기</BtnTitle>
```

스타일 컴포넌트에서는 nesting 기능을 최대한 지양해주세요!
버튼 컴포넌트 안에 span을 사용하는데 스타일 차이가 color 하나 뿐이라서
마치 컴포넌트의 Props를 사용하는 방법처럼 스타일을 구현하는 방법이 있습니다!
아래 내용을 참고해주세요!